### PR TITLE
Delete session file on json-parse-error

### PR DESCRIPTION
**Pull request by LopatkinEvgeniy**
Once i have problem with json-parse error. For some reason session file was empty and this error was repeated endlessly. In this fix i just remove bad session file when this error occurs.

*_PS sorry for poor english_

### CODE REVIEW

**Comment by valery-barysok** [comment]
i merged this PR but there is issue with reattempt logic (see session-file-helpers.js:get). On other hand i think it is not really relevant functionality now because we write into the temp file and then replace origin file as atomic update.

File: lib/session-file-helpers.js [hunk 1]
```diff
@@ -146,8 +146,14 @@ var helpers = {
           var json;
           try {
             json = JSON.parse(options.encrypt ? helpers.decrypt(data, sessionId) : data);
-          } catch (err2) {
-            err = err2;
+          } catch (parseError) {
+            return fs.remove(sessionPath, function (removeError) {
```
Comment (valery-barysok) [inline lib/session-file-helpers.js]: breaks reattempt logic

**Comment by LopatkinEvgeniy** [comment]
Add this changes in npm please.

**Comment by valery-barysok** [comment]
will do it on weekend. sorry for delay.

**Comment by valery-barysok** [comment]
done. see 0.2.2 but this PR does not complete yet and required to solve issue above.

**Pull request merged**